blazor/file-upload/how-to/resizing-the-image-before-sending-to-the-server.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: Resizing the Image Before Sending to the Server | Syncfusion
+description: Learn here all about Resizing the Image Before Sending to the Server in Blazor File Upload Component and more details.
+platform: Blazor
+control: File Upload
+documentation: ug
+---
+
+# Resizing the Image Before Sending to the Server
+
+To resize an image before sending the uploaded file to the server, you can utilize an Image instance. The provided code demonstrates how to take an uploaded image, resize it to the desired dimensions, and save it to a specified file path. This ensures that the image is adjusted to fit the required width and height before being stored. For your convenience.
+
+
+{% tabs %}
+{% highlight razor %}
+
+@using SixLabors.ImageSharp;
+@using SixLabors.ImageSharp.Processing;
```

Comment:
remove unwanted references, 
provide details about the use of third party library